packages/@okta/vuepress-site/docs/concepts/brands/index.md

```diff
@@ -40,15 +40,19 @@ Okta Verify and the Okta plugin support multibrand. Use the following minimum ve
 
 You can create up to three custom domains with multibrand customizations. Increase your limit to 200 custom domains by contacting support.
 
-You can only visit a branded touchpoint (such as a logo, color, favicon, or an image applied to a page or email) after you map to a custom domain. Create a brand and map it to a custom domain. Then, you can make further customizations, preview them, and publish them. See [Custom domains](/docs/guides/custom-url-domain/main/#about-okta-domain-customization).
+You can only visit a branded touchpoint (such as a logo or color) after you map to a custom domain. Create a brand and map it to a custom domain. Then, you can make further customizations, preview them, and publish them. See [Custom domains](/docs/guides/custom-url-domain/main/#about-okta-domain-customization).
 
 ### About subdomain brands and custom brands
 
-Multibrand orgs have a non-deletable default brand called the subdomain brand. However, you can create several custom brands. The subdomain brand always appears at the Okta subdomain URL and can’t have a custom domain. You can swap out the logo, background image, favicon, colors, and edit emails, but you can’t edit custom code for the sign-in page or error pages. You can only use custom domain and custom code for sign-in page and error-pages on custom brands.
+Multibrand orgs have a non-deletable default brand called the subdomain brand. However, you can create several custom brands. The subdomain brand always appears at the Okta subdomain URL and can’t have a custom domain. You can swap out the logo and other assets, but you can’t edit custom code for the sign-in page or error pages. You can only use custom domain and custom code for sign-in page and error-pages on custom brands.
 
-### Authentication flow
+### Multibrand customizations and emails
```

Comment:
@snehaarvindmani-okta @gormanho-okta sorry. this paragraph.

Reply:
LGTM 👍

Reply:
Thanks!